### DISCUSSION TITLE
Docker compose version matching

### DISCUSSION THREAD

**Discussion by cryptolukas** [renovatebot/renovate#30941] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitea Actions with ghcr.io/renovatebot/renovate:38.39.1

### Please tell us more about your question or problem

I have activated renovate for docker-compose updates.
For normal semver it works fine but does it work for docker tags like `16.4-alpine` with an appendix?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Yes
